build.gradle

```diff
@@ -0,0 +1,78 @@
+plugins {
+	id 'java'
+	id 'org.springframework.boot' version '2.3.3.RELEASE'
+	id 'io.spring.dependency-management' version '1.0.10.RELEASE'
+    id 'org.jetbrains.kotlin.jvm' version '1.3.72'
```

Comment:
I feel like a lot of indentation is off here - are we going with 2 spaces or 4 spaces for backend indents? We've got 2 spaces for the frontend, but I know 4 can be more readable for the backend code 🤷

Reply:
The build.gradle is weird because the programming language is groovy which has weird indents. I'll try and stay consistent going forward in this file - but this is one file that's going to be weird and different because it's one of the only groovy scripts we'll have and it comes with it's own weird syntax.